src/materialized/src/bin/materialized/main.rs

```diff
@@ -379,7 +381,7 @@ environment:{}",
         env_message
     );
 
-    adjust_rlimits();
+    sys::adjust_rlimits();
```

Comment:
Is this commit mainly to organize this crate better and not have it balloon into one giant file? If not, then I'm not sure I understand the purpose for this commit

Reply:
Yep, exactly!